[2025-03-25T07:43:11.098+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-25T07:43:11.145+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_2.first_task scheduled__2025-03-24T00:00:00+00:00 [queued]>
[2025-03-25T07:43:11.160+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_2.first_task scheduled__2025-03-24T00:00:00+00:00 [queued]>
[2025-03-25T07:43:11.160+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-25T07:43:11.191+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-03-24 00:00:00+00:00
[2025-03-25T07:43:11.209+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_2', 'first_task', 'scheduled__2025-03-24T00:00:00+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/fristdags.py', '--cfg-path', '/tmp/tmpyn2_k5b6']
[2025-03-25T07:43:11.205+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1207) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-25T07:43:11.216+0000] {standard_task_runner.py:105} INFO - Job 64: Subtask first_task
[2025-03-25T07:43:11.216+0000] {standard_task_runner.py:72} INFO - Started process 1209 to run task
[2025-03-25T07:43:11.378+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_2.first_task scheduled__2025-03-24T00:00:00+00:00 [running]> on host 7769c9b87de6
[2025-03-25T07:43:11.565+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_2' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-24T00:00:00+00:00'
[2025-03-25T07:43:11.567+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-25T07:43:11.677+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-25T07:43:11.698+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'hello this first task'"]
[2025-03-25T07:43:11.778+0000] {subprocess.py:99} INFO - Output:
[2025-03-25T07:43:11.851+0000] {subprocess.py:106} INFO - hello this first task
[2025-03-25T07:43:11.857+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-25T07:43:11.975+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-25T07:43:11.976+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_2, task_id=first_task, run_id=scheduled__2025-03-24T00:00:00+00:00, execution_date=20250324T000000, start_date=20250325T074311, end_date=20250325T074311
[2025-03-25T07:43:12.141+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-25T07:43:12.164+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
